pkg/storage/allocator_test.go

```diff
@@ -1913,6 +1924,79 @@ func TestAllocatorComputeAction(t *testing.T) {
 	}
 }
 
+func TestAllocatorComputeActionDisableStatsRebalance(t *testing.T) {
```

Comment:
This test needs to set the `EnableStatsBasedRebalancing` cluster setting to true to be interesting at all.

Reply:
Done.